src/proposals/20250417/SparkEthereum_20250417.t.sol

```diff
@@ -0,0 +1,180 @@
+            newOptimal: 0.8e27,
+            newBase:    0,
+            newSlope1:  0.01e27,
+            newSlope2:  3e27
+        });
+    }
+
+    function test_ETHEREUM_TBTCChanges() public onChain(ChainIdUtils.Ethereum()) {
+        _testIRMChanges({
+            asset:      Ethereum.TBTC,
+            oldOptimal: 0.6e27,
+            oldBase:    0,
+            oldSlope1:  0.04e27,
+            oldSlope2:  3e27,
+            newOptimal: 0.8e27,
+            newBase:    0,
+            newSlope1:  0.01e27,
+            newSlope2:  3e27
+        });
+    }
+
+    function test_ETHEREUM_WBTCChanges() public onChain(ChainIdUtils.Ethereum()) {
+        loadPoolContext(_getPoolAddressesProviderRegistry().getAddressesProvidersList()[0]);
+
+        ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot('', pool);
+        ReserveConfig memory config = _findReserveConfigBySymbol(allConfigsBefore, 'WBTC');
+
+        assertEq(config.liquidationThreshold, 50_00);
+
+        executeAllPayloadsAndBridges();
+
+        ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot('', pool);
+        
+        config.liquidationThreshold = 45_00;
+
+        _validateReserveConfig(config, allConfigsAfter);
+    }
+
+    function test_ETHEREUM_SUSDSChanges() public onChain(ChainIdUtils.Ethereum()) {
+        loadPoolContext(_getPoolAddressesProviderRegistry().getAddressesProvidersList()[0]);
+
+        ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot('', pool);
+        ReserveConfig memory config = _findReserveConfigBySymbol(allConfigsBefore, 'sUSDS');
+
+        assertEq(config.eModeCategory, 0);
+
+        executeAllPayloadsAndBridges();
+
+        ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot('', pool);
+        
+        config.eModeCategory = 2;
+
+        _validateReserveConfig(config, allConfigsAfter);
+    }
+
+    function test_ETHEREUM_morpho_PTSUSDE31JUL2025Onboarding() public onChain(ChainIdUtils.Ethereum()) {
+        _testMorphoCapUpdate({
+            vault: Ethereum.MORPHO_VAULT_DAI_1,
+            config: MarketParams({
+                loanToken:       Ethereum.DAI,
+                collateralToken: PT_SUSDE_31JUL2025,
+                oracle:          PT_SUSDE_31JUL2025_PRICE_FEED,
+                irm:             Ethereum.MORPHO_DEFAULT_IRM,
+                lltv:            0.915e18
+            }),
+            currentCap: 0,
+            newCap:     400_000_000e18
+        });
+        _testMorphoPendlePTOracleConfig({
+            pt:           PT_SUSDE_31JUL2025,
+            oracle:       PT_SUSDE_31JUL2025_PRICE_FEED,
+            discount:     0.2e18,
+            currentPrice: 0.939981424403855911e36
+        });
+    }
+
+    function test_ARBITRUM_FluidSUSDSOnboarding() public onChain(ChainIdUtils.ArbitrumOne()) {
+        _testERC4626Onboarding({
+            vault:                 ARBITRUM_FLUID_SUSDS,
+            expectedDepositAmount: 5_000_000e18,
```

Comment:
Why do we choose this value?
the same value was in spell 20250320, but `depositMax` and `depositSlope` were higher than here

Reply:
It's a bit arbitrary, but it's meant to be a "reasonable" value we will use to allocate. It's possible there is some rate limits on the protocol itself so we can't assume this value is related to the rate limits we impose.